modules/tinymce/src/plugins/template/demo/html/development.html

```diff
@@ -2,8 +2,8 @@
 <div class="mceTmpl">
   <table width="98%"  border="1" cellspacing="0" cellpadding="0">
     <tr>
-      <th scope="col">Header 1</th>
-      <th scope="col">Header 2</th>
+      <th scope="col">Heading 1</th>
```

Comment:
Another place that probably shouldn't have been changed?

Reply:
```suggestion
      <th scope="col">Header 1</th>
```